utils/xjson.py

```diff
@@ -88,29 +87,9 @@ def rsorter(o):
                 return new_dict
 
             elif isinstance(o, set):
```

Comment:
Why are you also changing set? sets are most certainly unordered?

Reply:
I was still sorting them, but I was sorting using `<` rather than `key < key`, fixed.

Reply:
Oh, I rewrote this to drop the dependency on `OrderedSet`.

Reply:
LGTM.